[2025-04-09T05:01:05.579+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T05:01:05.604+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-09T05:00:31.395948+00:00 [queued]>
[2025-04-09T05:01:05.614+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-09T05:00:31.395948+00:00 [queued]>
[2025-04-09T05:01:05.615+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-09T05:01:05.628+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): meteo_operations.collect_meteo_data> on 2025-04-09 05:00:31.395948+00:00
[2025-04-09T05:01:05.639+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.collect_meteo_data', 'manual__2025-04-09T05:00:31.395948+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/operators/testdag.py', '--cfg-path', '/tmp/tmpxodj0d26']
[2025-04-09T05:01:05.641+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask meteo_operations.collect_meteo_data
[2025-04-09T05:01:05.643+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=260) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-09T05:01:05.643+0000] {standard_task_runner.py:72} INFO - Started process 278 to run task
[2025-04-09T05:01:05.692+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-09T05:00:31.395948+00:00 [running]> on host bb94a3d1492a
[2025-04-09T05:01:05.767+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.collect_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T05:00:31.395948+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T05:00:31.395948+00:00'
[2025-04-09T05:01:05.768+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T05:01:05.804+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 21.4901, 'longitude': 39.1862, 'timestamp': '2025-04-09 00:00:00', 'temperature': 26.323501586914062, 'humidity': 68.0, 'wind_speed': 4.104631423950195, 'pm2_5': 100.0999984741211, 'co': 281.0}
[2025-04-09T05:01:05.828+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T05:01:05.829+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.collect_meteo_data, run_id=manual__2025-04-09T05:00:31.395948+00:00, execution_date=20250409T050031, start_date=20250409T050105, end_date=20250409T050105
[2025-04-09T05:01:05.871+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T05:01:05.897+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T05:01:05.899+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
